hawc/apps/lit/views.py

```diff
@@ -440,6 +440,14 @@ class RefEdit(BaseUpdate):
     form_class = forms.ReferenceForm
 
 
+class RefDelete(BaseDelete):
+    success_message = "Reference deleted."
+    model = models.Reference
+
+    def get_success_url(self):
+        return reverse_lazy("lit:overview", kwargs={"pk": self.assessment.pk})
```

Comment:
More recently, I'm using args instead of kwargs b/c it's shorter: `return reverse_lazy("lit:overview", args=(self.assessment.pk,))`